lib/cache/memory-cache-store.js

```diff
@@ -213,6 +213,20 @@ class MemoryCacheStore {
     this.#data.delete(origin)
   }
 
+  /**
+   * @param {string} origin
+   * @param {string} path
+   */
+  deleteByOriginAndPath (origin, path) {
+    const cachedPaths = this.#data.get(origin)
+    for (const key of cachedPaths.keys()) {
+      // not good to use startsWith here, but it's fine for now
+      if (key.startsWith(path)) {
```

Comment:
I would split the `key` here by the `:` because the `MemoryCacheStore` indeed stores keys in a format `path:method`, but this might be not true for other types of stores. So if someone will rely on this it might be a breaking change in the future.